modules/nf-core/jvarkit/vcfpolyx/main.nf

```diff
@@ -23,24 +23,28 @@ process JVARKIT_VCFPOLYX {
     task.ext.when == null || task.ext.when
 
     script:
-    def args1  = task.ext.args1 ?: ''
-    def args2  = meta.vcfpolyx_args ?: (task.ext.args2 ?: ' --tag POLYX --max-repeats 10 ')
-    def args3  = task.ext.args3 ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args1         = task.ext.args1 ?: ''
+    def args2         = task.ext.args2 ?: ' --tag POLYX --max-repeats 10 '
+    def args3         = task.ext.args3 ?: ''
+    def prefix        = task.ext.prefix ?: "${meta.id}"
+    def regions_file  = regions_file ? (tbi ? " --regions-file" : " --targets-file") + " \"${regions_file}\" " : ""
```

Comment:
Can't review properly, but a quick note, it is best not to override the input variables.
```suggestion
    def regions_file_command  = regions_file ? (tbi ? " --regions-file" : " --targets-file") + " \"${regions_file}\" " : ""
```
Are the slashes at the end to escape the "? In which case you could just use ' for one of the sets of quotes.